Makefile

```diff
@@ -28,6 +28,17 @@ test:
 	go get gotest.tools/gotestsum
 	go run gotest.tools/gotestsum -- -timeout 1200s -parallel 5 ./... $(ARGS)
 
+.PHONY: test-end-to-end
+test-end-to-end: install
+	cd end-to-end-test/ && $(MAKE)
+
+.PHONY: test-cog-library
+test-cog-library:
+	cd pkg/docker/ && pytest cog_test.py
+
+.PHONY: test-all
+test-all: test test-end-to-end test-cog-library
```

Comment:
`make test` should probably test everything, right?

Reply:
Yep, done. The old `make test` is now `make test-go`.